src/docfx/lib/ArrayBuilder.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+namespace System.Collections.Concurrent
+{
+    internal class ArrayBuilder<T> : IEnumerable<T>
```

Comment:
```suggestion
    internal readonly struct ArrayBuilder<T>
```

Reply:
use `class` because `struct` doesnt support parameterless constructor and field initialization